labrad/protocol.py

```diff
@@ -391,10 +405,14 @@ def connect(host=C.MANAGER_HOST, port=None, tls_mode=C.MANAGER_TLS):
     if port is None:
         port = C.MANAGER_PORT_TLS if tls_mode == 'on' else C.MANAGER_PORT
 
+    def done(p):
+        p.set_address(host, port)
+        returnValue(p)
+
```

Comment:
twisted doesn't recommend  calling returnValue from inside a subfunction of `@inlinecallbacks`.  and says `done()` should be a generator as well.  https://twistedmatrix.com/documents/current/api/twisted.internet.defer.html#returnValue